lib/dao.js

```diff
@@ -1899,7 +1918,6 @@ DataAccessObject.prototype.save = function (options, cb) {
     Model.applyProperties(data, inst);
     inst.setAttributes(data);
 
-    // validate first
```

Comment:
I don't see why this comment is no longer relevant? Would you mind putting it back again?